### DISCUSSION TITLE
[Feature Request]: Add information within manual how to interact with HA Energy panel

### DISCUSSION THREAD

**Discussion by jacky-coke** [wlcrs/huawei_solar#437] [Setting up the integration] [answered]
### Describe your feature request

Dear team, 
I really appreciate this integration and it's working great for me! 
For only a couple of Euros I bought a TP-Link Wifi Client device, I connected it to my SUN2000 and it forwards all information through my home network into my NAS, hosting the HA VM. 

I'm wondering which sensors to assign within the 'energy'-module of HA and was wondering that this can't be found within your manual (or maybe I'm blind?!). 
<img width="740" alt="image" src="https://github.com/wlcrs/huawei_solar/assets/80129168/55a80e6c-6b74-41b8-ac10-22316b4398b7"> (sorry, only german screenshot available). 

Any chance to add this? 
Best regards, 
Nik

### Proper usage

- [X] I confirm that this is not a bug report or support request
- [X] I confirm that this feature request is within the stated scope of the integration

**Comment by nurzen** [comment, marked as answer]
you need an extra meter for this, its not coming from the inverter. It`s feed an consumption to the net. You can use an ir dongle to read that measure from the meter your provider gives you (assuming its a relative new meter) or use one extra meter like Huawei dtsu666-h.
[https://wiki.volkszaehler.org/hardware]

HA uses this information to calculate your own consumption. The difference between what your inverter produces and what you feed to the net is the energy you consume from your solar panels (Eigenverbrauch, Eigenverbrauchsquote).

The information from the inverter go to a different panel.
![Screenshot 2023-08-02 110428](https://github.com/wlcrs/huawei_solar/assets/8253296/16ab5830-bc6c-481c-9194-fd9011b77b37)
